### ISSUE TITLE
Fine tuning on custom number of classes

### ISSUE THREAD

**Issue by deval-mehta** [monologg/GoEmotions-pytorch#3]
Let's say we want to fine-tune the model (any of the taxonomies - ekman/original) on another dataset having a different number of classes for e.g. only positive and negative. Then what's the correct procedure to do it?

Currently, If I prepare the data in the same format (.tsv files with data and labels), and put the `labels.txt` as having only two classes, the training seems to run. But is this way correct? Or any other changes need to be made inside the model training?
